readthedocs/projects/views/private.py

```diff
@@ -506,19 +506,25 @@ def project_notifications(request, project_slug):
     project = get_object_or_404(
         Project.objects.for_admin_user(request.user), slug=project_slug)
 
-    email_form = EmailHookForm(data=request.POST or None, project=project)
-    webhook_form = WebHookForm(data=request.POST or None, project=project)
+    project_dashboard = reverse('projects_notifications', args=[project.slug])
 
-    if request.method == 'POST':
+    if request.method == 'POST' and 'email' in request.POST:
```

Comment:
I think the logic that was written before was easier to read. Create both forms, if the email one is valid, save it. Otherwise, if the webhook form is valid, save it.

This checking of `email in request.POST` is easier to break. I prefer to change it back as it was.

Reply:
Okay, I will do that. :+1: